website/docs/docs/cloud/manage-access/set-up-sso-azure-active-directory.md

```diff
@@ -41,13 +41,13 @@ need to select the appropriate directory and then register a new application.
 
 4. Configure the **Redirect URI**. The table below shows the appropriate
    Redirect URI values for single-tenant and multi-tenant deployments. For most
-   enterprise use-cases, you will want to use the single-tenant Redirect URI. Replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
+   enterprise use-cases, you will want to use the single-tenant Redirect URI. Replace `YOUR_AUTH_URL` with the [appropriate Auth URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
 
 
 | Application Type | Redirect URI |
 | ----- | ----- |
-| Single-Tenant _(recommended)_ | `https://YOUR_ACCESS_URL/complete/azure_single_tenant` |
-| Multi-Tenant | `https://YOUR_ACCESS_URL/complete/azure_multi_tenant` |
+| Single-Tenant _(recommended)_ | `https://YOUR_AUTH_URL/login/callback?connection=<login slug>` |
+| Multi-Tenant | `https://YOUR_AUTH_URL/login/callback?connection=<login slug>` |
```

Comment:
I'm pretty sure these both don't need the `connection` parameter either, but will confirm.